src/EnergyPlus/HeatBalFiniteDiffManager.cc

```diff
@@ -1470,6 +1497,8 @@ namespace HeatBalFiniteDiffManager {
 		if ( surface_ExtBoundCond == Ground || IsRain ) {
 			TDT( i ) = TT( i ) = TempOutsideAirFD( Surf );
 			RhoT( i ) = RhoVaporAirOut( Surf );
+			SurfaceFD( Surf ).CpDelXRhoS1( i ) = 0.0; // Outside face  does not have an outer half node
```

Comment:
@Myoldmopar In these routines, only one pair of these lines will execute for each call to the function  (ExteriorBCEqns, InteriorNodeEqns, IntInterfaceNodeEqns, or InteriorBCEqns).  So, I could 
  1. leave as-is, 
  2.  add a lookup to surfaceFD in each function that then gets used twice, or 
  3.  I could add two more arguments to each of these calls in CalcHeatBalFiniteDiff] and add lookups like [these existing ones for other variables in surfaceFD](https://github.com/NREL/EnergyPlus/blob/548ae8acdda3b697aa42ed60f52dfc5975b61b45/src/EnergyPlus/HeatBalFiniteDiffManager.cc#L1104-L1117).

Which of these would be most efficient?  Or some other option?